package.json

```diff
@@ -50,14 +51,23 @@
   },
   "homepage": "https://threejs.org/",
   "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-preset-env": "^1.7.0",
     "concurrently": "^4.0.1",
     "electron": "^2.0.8",
     "eslint": "^5.5.0",
     "eslint-config-mdcs": "^4.2.3",
     "google-closure-compiler": "20180805.0.0",
     "http-server": "^0.11.1",
-    "qunit": "^2.6.2",
```

Comment:
Why do you downgrade `qunit` and `rollup` version?

Reply:
Actually, the qunit  and rollup is latest version in this PR now. I made a mistake at first.( I created the branch based on master ) and after I PR, I found the conflicts and fixed it. So they are latest version now.